Credify/Credify/WebView/WebViewController.swift

```diff
@@ -191,7 +195,7 @@ extension WebViewController: WKNavigationDelegate {
     
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
         // NOTE: this is not working well
```

Comment:
This is not working well??

Reply:
It's your comment. But I removed it because of it's working weill